packages/kotti-single-select/src/SingleSelect.vue

```diff
@@ -236,15 +232,21 @@ export default {
 			if (!this.isOptionAllowed(option)) return
 
 			this.selectedLabel = option.label
-
-			const selectedItem = this.options.find(
-				({ label }) => label === this.selectedLabel,
-			)
+			// const selectedItem = this.options.find(
+			// 	({ label }) => label === this.selectedLabel,
+			// )
```

Comment:
Is that comment something you want to keep in the future?

Reply:
well it's now done by directly assigning the option to the selectedItem.. i checked that they are always the same object, but i wasn't sure if there was maybe a use-case in particular that would require it to be this way - so i left it just in case